packages/next-swc/crates/core/src/styled_jsx/transform_css.rs

```diff
@@ -162,88 +156,177 @@ struct Namespacer {
 
 impl VisitMut for Namespacer {
     fn visit_mut_complex_selector(&mut self, node: &mut ComplexSelector) {
+        #[cfg(debug_assertions)]
+        let _tracing = {
```

Comment:
This block explictly uses `#[cfg(debug_assertions)]` because it does some operation which cannot removed by `tracing`.